### DISCUSSION TITLE
`.top` 域名为何没法正常显示应用图标？

### DISCUSSION THREAD

**Discussion by TonyKingWJX** [lobehub/lobehub#2828] [General Question] [answered]
我将lobechat作为一个浏览器应用安装，但是安装以后会丢失图标，并且启动时还会黑屏。
![PixPin_2024-06-09_23-34-27](https://github.com/lobehub/lobe-chat/assets/115714101/76b1dfaf-27c8-44af-bad4-48bcd72b5076)
![PixPin_2024-06-09_23-35-15](https://github.com/lobehub/lobe-chat/assets/115714101/451f7cc2-a718-4752-a2b1-8a1b35c5d094)

**Comment by arvinxx** [comment]
你的域名是 `.top` 吗？

**Reply by TonyKingWJX** [reply]
是的，我的域名是.top。这个还和域名有关？

**Reply by arvinxx** [reply, marked as answer]
域名被CDN墙了，没解决方案。建议换一个后缀的域名
